src/components/binding/binding-loader.js

```diff
@@ -31,16 +32,23 @@ genotet.BindingLoader.prototype.LOCUS_MARGIN_RATIO = .1;
  * @param {string} fileName Binding file name.
  * @param {string} bedName Bed name of the bed binding track.
  * @param {string} chr ID of the chromosome.
+ * @param {number} numberOfTracks Number of tracks.
  * @param {number=} opt_track Track # into which the data is loaded.
  * @override
  */
 genotet.BindingLoader.prototype.load = function(fileName, bedName, chr,
```

Comment:
Yes. However, user can choose data for each track as they want in panel after loading. I think loading the same data at first is not that bad. Let me know if you have other better idea.